CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # Changelog
 
+## [1.3](https://github.com/arborx/arborx/tree/v1.3) (2022-10-12)
+
+[Full Changelog](https://github.com/arborx/arborx/compare/v1.2...v1.3)
+
+**New features:**
+- Add ordered tree traversal (EXPERIMENTAL) [\#683](https://github.com/arborx/ArborX/pull/683)
+- Add multi-dimensional support (EXPERIMENTAL) [\#713](https://github.com/arborx/ArborX/pull/713), [\#722](https://github.com/arborx/ArborX/pull/722), [\#726](https://github.com/arborx/ArborX/pull/726), [\#731](https://github.com/arborx/ArborX/pull/731)
+
+**Build changes:**
+- Require C++17 [\#715](https://github.com/arborx/ArborX/pull/715)
+- Require Kokkos 3.6 [\#748](https://github.com/arborx/ArborX/pull/748)
+
+**Enhancements:**
+- Switch to using stackless spatial traversal for all backends [\#672](https://github.com/arborx/ArborX/pull/672)
+- Add a ray-tracing example using ordered tree traversal [\#691](https://github.com/arborx/ArborX/pull/691)
+- Add `distance()` function to the nearest predicate [\#700](https://github.com/arborx/ArborX/pull/700)
+- Template `BruteForce` on bounding volume [\#712](https://github.com/arborx/ArborX/pull/712)
+- Warn about a potential loss of precision when using FDBSCAN-DenseBox [\#560](https://github.com/arborx/ArborX/pull/560)
+- Migrate DBSCAN driver from `examples/` to `benchmarks/` [\#736](https://github.com/arborx/ArborX/pull/736)
```

Comment:
Not an enhancement and we should probably add a proper example before we release